for-posdealers/02-pre-sales/02-purchase.md

```diff
@@ -10,14 +10,26 @@ Einen Überblick über den gesamten Prozess beginnend von der Registrierung im P
 
 siehe [Allgemeine Voraussetzungen für den Einkauf von Produkten](voraussetzungen-einkauf.md)
 
-Empfehlungen
+**Empfehlungen**
 
-- Kontrolle der Stammdaten auf Adresse und USt-Id Nummer für die Rechnungserstellung
+- Kontrollieren Sie vor der Bestellung Ihre Stammdaten auf Adresse und USt-Id Nummer für die Rechnungserstellung
 
-Schritt 1: Erwerb der Händlereinkaufsprodukte
+### Schritt 1: Wahl der Produkte
+
+- Wählen Sie im Portal im Menüpunkt "Shop/Products" Ihre gewünschten Produkte und füge Sie sie dem Warenkorb hinzu, oder
+- Wählen Sie im Portal im Menüpunkt "Shop/Quotes" ein für Sie erstelltes Angebot aus und fügen Sie es dem Warenkorb hinzu
+
+### Schritt 2: Checkout
+
+- Falls Sie auf Basis der Rahmenvereinbarung Anspruch auf Mengenrabatte haben, prüfen Sie ob der dargestellte Preis mit der Vereinbarung Ihres Rahmenvertrages übereinstimmt.
+- Prüfen Sie die Rechnungsdaten und die Lieferadresse.
+- Prüfen Sie die USt-IdNr auf Korrektheit. In Deutschland steht eine Schnittstelle zur Validierung der USt-IdNr aus datenschutzrechtlichen Gründen nicht zur Verfügung. Ist die USt-IdNr korrekt, fahren Sie bitte fort, ansonsten korrigieren Sie bitte Ihre Daten unter "Company/Master Data".
+- Wenn der Bestell-Button deaktiviert ist, kontaktieren Sie das Sales-Team um Ihr Kreditlimit erhöhen zu lassen
+- Mit Abschluss der Bestellung erhalten Sie die Rechnung per E-Mail an die im Primär-Kontakt ihres Accounts angegebene E-Mail Adresse zugestellt.
+
+### Schritt 3: Entitlement Verwaltung
+
+Im Shop können Sie unter dem Menüpunkt "Entitlements" werden Ihre Entitlements ("Ansprüche") für ihre erworbenen Produkte dargestellt. Diese Ansprüche auf Produkte können auf Betreiber übertragen und  Betreiber-Produkte kostenfrei im Shop ausgechecked werden.
```

Comment:
Proposal: Im Shop werden unter dem Menüpunkt "Entitlements" Ihre Entitlements ("Ansprüche") für die erworbenen Produkte dargestellt. Diese Ansprüche auf Produkte können auf Betreiber übertragen, und in der Folge kostenfrei für Produkte eingelöst werden.

Reply:
cool, danke dir!